src/collection-view.js

```diff
@@ -99,7 +99,7 @@ const CollectionView = Backbone.View.extend({
   _initialEvents() {
     if (this.collection) {
       this.listenTo(this.collection, 'add', this._onCollectionAdd);
```

Comment:
sounds good.  That was my first comment, but this is already a sizable PR so it makes sense to wait probably anyhow. 👏